templates/todo/web/react-fluent/package.json

```diff
@@ -17,7 +17,7 @@
     "prestart": "npm run envconfig",
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "postbuild": "shx cp ./entrypoint.sh ./build",
+    "postbuild": "shx cp ./entrypoint.sh ./build && ls staticwebapp.config.json && cp staticwebapp.config.json ./build || true ",
```

Comment:
This will likely fail on `windows` and you'll likely need to use the `shx` lib that I am using before copying the entrypoint.

Reply:
This is using the same `shx` ,   shx supports creating a chain of calls just like bash, where `&&` invokes the next command if the previous command returns error code == 0.  Otherwise it calls what's set with `||`